src/app/api/v1/admin/notifications/route.ts

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { NotificationRow } from "knex/types/tables";
+import { NextRequest, NextResponse } from "next/server";
+import createDbConnection from "../../../../../db/connect";
+import { getAllNotifications } from "../../../../../db/tables/notifications";
+import { getServerSession } from "../../../../functions/server/getServerSession";
+import { isAdmin } from "../../../utils/auth";
+import { logger } from "../../../../functions/server/logging";
+
+const knex = createDbConnection();
+
+export async function GET() {
+  const session = await getServerSession();
+  console.log("session from default route: ", session);
+
+  if (!isAdmin(session?.user?.email || "")) {
+    logger.error("Not an admin user");
```

Comment:
Is it OK that this continues or did you want to return `401` here? If it's OK then it shouldn't be `error` severity log